rascaline/src/calculators/radial_basis/mod.rs

```diff
@@ -28,7 +28,22 @@ pub enum RadialBasis {
     /// The easiest way to create a set of spline points is the
     /// `rascaline.generate_splines` Python function.
     TabulatedRadialIntegral {
+        /// spline points
         points: Vec<SplinePoint>,
+        /// Contribution of the central atom according to
+        ///
+        ///   $$ \langle n000 \vert g \rangle =
+        ///        \sqrt{4 \pi} \int \mathrm{d}r r^2 R_n(r) g(r)$$
+        ///
+        /// where $g(r)$ is a radially symmetric density function, $R_n(r)$ the radial
+        /// basis function and $n$ the current radial channel. Note that the integration
+        /// range was deliberately left ambiguous since it depends on the radial basis,
+        /// i.e. for the GTO basis, $r \in \mathbb{R}^+$ is used, while
+        /// $r \in [0, r_\mathrm{cut}]$ for the monomial basis.
+        ///
+        /// This parameter is only required for LODE calculations where the length of
+        /// the provided `center_contribution` must be the same as `max_radial`.
```

Comment:
Unfortunately, we don't render inner members documentation in the doc, could you move this above in the `TabulatedRadialIntegral` doc? Cf the link in my other comment for a rendered version.